core/Kconfig

```diff
@@ -45,4 +45,10 @@ config MOD_CORE_PANIC
 config MOD_CORE_THREAD_FLAGS
     bool "Thread flags"
 
+config THREAD_NAMES
```

Comment:
The symbol looks good. But it's in the middle of a menu defined only for migration. I would suggest something like the following, placed after the `endif`:

```
menuconfig KCONFIG_MODULE_CORE
    bool "Configure RIOT Core"
    depends on MODULE_CORE
    help
        Configure RIOT Core using Kconfig.

if KCONFIG_MODULE_CORE

config THREAD_NAMES
    bool "Store thread name strings"
    help
        By default, thread names are not stored if DEVELHELP is not used.
        Use this parameter to store them for non-devel builds.

endif
```

Reply:
done. Thanks for the tip